src/utils/index.js

```diff
@@ -214,3 +214,31 @@ export function setTheme(selectedTheme = getTheme()) {
   );
   localStorage.setItem('tkn-theme', theme);
 }
+
+function sanatizeCancelState(state) {
+  const valid = [
+    'Cancelled',
+    'CancelledRunFinally',
+    'StoppedRunFinally'
+  ].includes(state);
+  if (valid) {
+    return state;
+  }
+  return 'PipelineRunCancelled';
```

Comment:
We have a different default here vs.  the end of `getDefaultCancelState` below which calls this function. Should they be consistent / combined?